ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -71,45 +77,25 @@ private[apiserver] final class ApiTransactionService private (
       logging.endInclusive(request.endInclusive),
       logging.parties(request.filter.filtersByParty.keys),
     ) { implicit loggingContext =>
-      val subscriptionId = subscriptionIdCounter.incrementAndGet().toString
-      logger.info(s"Received request for transaction subscription $subscriptionId: $request")
+      logger.info("Received request for transactions.")
+      logger.trace(s"Transaction request: $request")
       transactionsService
         .transactions(request.startExclusive, request.endInclusive, request.filter, request.verbose)
         .via(logger.debugStream(transactionsLoggable))
         .via(logger.logErrorsOnStream)
         .via(StreamMetrics.countElements(metrics.daml.lapi.streams.transactions))
     }
 
-  private def transactionsLoggable(transactions: GetTransactionsResponse): String =
-    s"Responding with transactions: ${transactions.transactions.toList
-      .map(t => entityLoggable(t.commandId, t.transactionId, t.workflowId, t.offset))}"
-
-  private def transactionTreesLoggable(trees: GetTransactionTreesResponse): String =
-    s"Responding with transaction trees: ${trees.transactions.toList
-      .map(t => entityLoggable(t.commandId, t.transactionId, t.workflowId, t.offset))}"
-
-  private def entityLoggable(
-      commandId: String,
-      transactionId: String,
-      workflowId: String,
-      offset: String,
-  ): LoggingEntries =
-    LoggingEntries(
-      logging.commandId(commandId),
-      logging.transactionId(transactionId),
-      logging.workflowId(workflowId),
-      "offset" -> offset,
-    )
-
   override def getTransactionTrees(
       request: GetTransactionTreesRequest
-  ): Source[GetTransactionTreesResponse, NotUsed] =
+  ): Source[GetTransactionTreesResponse, NotUsed] = {
     withEnrichedLoggingContext(
       logging.startExclusive(request.startExclusive),
       logging.endInclusive(request.endInclusive),
       logging.parties(request.parties),
     ) { implicit loggingContext =>
-      logger.info(s"Received request for transaction tree subscription: $request")
+      logger.info("Received request for transaction trees.")
+      logger.trace(s"Transaction tree request: $request")
```

Comment:
Nit: for consistency, you might want to use the same pattern for all log messages, e.g., "\<what\> request: \<actual-request\>". None of the below log messages follow the pattern for this particular line.

Reply:
Whoops, I seem to have deleted the word "request" half-way down. I'll add it back in.